src/app/swap/swap-out.ts

```diff
@@ -48,16 +48,17 @@ export const swapOut = async (): Promise<SwapOutResult | SwapServiceError> => {
     },
   })
 
-  if (swapOutAmount instanceof Error) return swapOutAmount
+  const swapNoOp: SwapOutResult = {
+    noOp: true,
+    htlcAddress: "" as OnChainAddress,
+    serverMessage: "",
+    swapId: "" as SwapId,
+    swapIdBytes: "",
+  }
 
-  if (swapOutAmount.amount === 0n)
-    return {
-      noOp: true,
-      htlcAddress: "" as OnChainAddress,
-      serverMessage: "",
-      swapId: "" as SwapId,
-      swapIdBytes: "",
-    }
+  if (swapOutAmount instanceof SwapErrorNonCritical) return swapNoOp
```

Comment:
@nicolasburtey  this is where I handle a non-critical error and instead of returning an instance of an Error (which throws from the caller...i.e cron) i instead return a NoOp SwapOutResult

Reply:
In places like this in other use-cases we would also simply return something like `false`. Having errors carry context like this could be tricky.

Reply:
I made it a NoOp swap result based on this feedback from Justin https://github.com/GaloyMoney/galoy/pull/1396#discussion_r973624924